pkg/rpc/server.go

```diff
@@ -209,6 +208,31 @@ Methods:
 			results = "Invalid assetid"
 		}
 
+	case "getaccountstate":
+		var err error
+
+		param, exists := reqParams.ValueAtAndType(0, "string")
+		if !exists {
+			err = errors.New("expected param at index 0 to be a valid string account address parameter")
+			resultsErr = NewInvalidParamsError(err.Error(), err)
+			break
+		}
+
+		scriptHash, err := crypto.Uint160DecodeAddress(param.StringVal)
+		if err != nil {
+			err = errors.Wrapf(err, "unable to decode %s to Uint160", param.StringVal)
+			resultsErr = NewInvalidParamsError(err.Error(), err)
+			break
+		}
+
+		as := s.chain.GetAccountState(scriptHash)
+
+		if as != nil {
+			results = wrappers.NewAccountState(as)
+		} else {
+			results = "Invalid public account address"
+		}
```

Comment:
how's that for a decision?
```go
		param, exists := reqParams.ValueAtAndType(0, "string")
		if !exists {
			err = errors.New("expected param at index 0 to be a valid string account address parameter")
			resultsErr = NewInvalidParamsError(err.Error(), err)
		} else if scriptHash, err := crypto.Uint160DecodeAddress(param.StringVal); err != nil {
			err = errors.Wrapf(err, "unable to decode %s to Uint160", param.StringVal)
			resultsErr = NewInvalidParamsError(err.Error(), err)
		} else if as := s.chain.GetAccountState(scriptHash); as != nil {
			results = wrappers.NewAccountState(as)
		} else {
			results = "Invalid public account address"
		}
```

Reply:
:+1:

Reply:
In that case, I suggest to correct thus

Reply:
@im-kulikov ,
I don't understand your comment. Could you please clarify? Is the logic to complicate?

Reply:
It makes no sense to call a break..

Reply:
I suggest to get rid thus of a call break

Reply:
Are looking at the latest commit (`cosmetic changes`)?  I think I have already got rid of all `break` calls.

Reply:
Oh.. yesterday when I looked, it was still with the old code..

Great 👍